### ISSUE TITLE
pow(0.5) is not working

### ISSUE THREAD

**Issue by huahuayu** [shopspring/decimal#222]
hi, pow(0.5) is not working that is say sqrt() is not working, I see someone has raised PR, why not merge it?

**Comment by mwoss** [comment]
Hi!
Yup, `Pow` still misfunction. As far as I remember I didn't merge PR with sqrt as it was missing unit tests. I can check if this functionality is working correctly and if so merge it to master and write tests itself. Will check in the free time.

**Comment by huahuayu** [comment]
how it's going?

**Comment by mwoss** [comment]
Sorry, I had no time to do anything with decimal last week. I will try to validate the correctness of implementation #130 tomorrow.

**Comment by huahuayu** [comment]
> Sorry, I had no time to do anything with decimal last week. I will try to validate the correctness of implementation #130 tomorrow.

don't worry, take your time.

**Comment by LexRiver** [comment]
do not work for me also

**Comment by mwoss** [comment]
Yes, that's right. This issue is still not fixed. I was working on that some time ago and I've already prepared an implementation of exp funcs #229 that will be used for proper Pow implementation, but I had to switch my focus lately on my Master Thesis.
I will get back to it after the defense.

**Comment by mwoss** [comment]
I've finally implemented an improved `Pow` method. Please take a look at this comment: https://github.com/shopspring/decimal/issues/55#issuecomment-2033204145
